### DISCUSSION TITLE
AlphaPulldown for protein > 2000 amino acids

### DISCUSSION THREAD

**Discussion by trangnguyen33333** [KosinskiLab/AlphaPulldown#157] [Q&A]
Hello, I used the AlphaPulldown and it worked well for any protein candidates with ~800 amino acids. However, for protein with 2400 amino acids, it took me 11 days and not yet finished. Is there anyone here ever using the AlphaPulldown with a very long protein,  and did it work? Thank you so much

**Comment by dingquanyu** [comment]
Hi,

Thanks for your interest in our package but unfortunately, the prediction time really depends on your GPU card, how deep the MSA etc. Could you let us know how many predictions per model have you set and how many predictions have been finished within 11 days?

Yours,
Dingquan

**Comment by trangnguyen33333** [comment]
Hi,
I did 1 prediction per model and 5 models in total. Specifically, we just had 32 CPUs maximum but I could boost up to 80 CPUs and this was limited in 4 days. Therefore, when I ran my bait and my candidate protein (2400 amino acids), I used 80 CPUs but because of the time limitation,  it was terminated after 4 days. After that, I lowered down to 32 CPUs which was limited up to 31 days. Unfortunately, there was no prediction finished within 11 days. Hope this help. Thanks a lot.



Best,
Trang

**Reply by superantichrist** [reply]
I think time increase exponentially with seq's length. I divide it to part of length 200~300 (according pae value or unitprot structure part) and it takes 3 min avg in 2080 ti gpu. Hope it helps.

**Reply by jkosinski** [reply]
Hi Trang,

AlphaFold and, therefore, AlphaPulldown is slow on CPU. You should use GPU instead. Still, for long bait and prey, you need a strong card and likely Nvidia unified memory. If you don't have that, chop the sequences as suggested above.

Best,
Jan
